.github/workflows/gradle.publish.yaml

```diff
@@ -146,6 +113,18 @@ jobs:
           onlyLastTag: true
           maxIssues: ${{ inputs.changelog_max_issues }}
 
+      - id: generate-release-information
+        name: "Generate release information"
+        run: |
+          release_version_suffix = "alpha"
+          if [[ ${{ inputs.curse_release_type }} == release ]]; then
+            release_version_suffix = ""
+          elif [[ ${{ inputs.curse_release_type }} == beta ]]; then
+            release_version_suffix = "snapshot"
+          fi
+          echo "Release version suffix: ${release_version_suffix}"
+          echo "release_version_suffix=${release_version_suffix}" >> "$GITHUB_ENV"
+
```

Comment:
Realistically this entire compute can trivially happen in OPC, which is much faster then this seperate step, but in reality I don't really care, as it for now implements the spec, and if I want to move it, I could easily do so.